skaffold.yaml

```diff
@@ -46,7 +64,7 @@ profiles:
         value:
           buildArgs:
             AEA_VERSION: "1.14.0"
-            AEA_AGENT: "{{.AEA_AGENT}}"
+            AEA_AGENT: "{{.AEA_AGENT_HASH}}"
```

Comment:
@DavidMinarsch We need an agent hash in here after each release. 
We could set it from an env var, I couldnt quite make the make command do it.

Reply:
Nice one liner